### DISCUSSION TITLE
Issue links redirecting to pulls broke

### DISCUSSION THREAD

**Discussion by Mm2PL** [community/community#8691] [Projects and Issues]
`https://github.com/owner/repo/issues/PR_NUMBER` links redirecting to `https://github.com/owner/repo/pulls/PR_NUMBER` automagically stopped working. I have functionality that relies on this behavior because it simplifies my app immensely. It's worth noting that issue -> discussion redirects still work. Can you please restore this functionality?

**Comment by Felanbird** [comment]
Also worth noting that the API still handles these redirects.
![image](https://user-images.githubusercontent.com/41973452/145652853-7cbbff08-9041-492e-a57b-1d8b4a27a8ee.png)

**Comment by Shabirmean** [comment]
Had the same issue last Friday, but they seems to be fine this morning!
